static/app/views/dashboards/widgetCard/widgetCardContextMenu.tsx

```diff
@@ -257,10 +257,10 @@ function WidgetCardContextMenu({
                 position="bottom-end"
                 disabledKeys={[...disabledKeys]}
               />
-              <OpenWidgetViewerButton
+              <Button
                 aria-label={t('Open Widget Viewer')}
-                priority="link"
-                size="zero"
+                borderless
+                size="xs"
```

Comment:
`priority="link"` makes the button look like a link (with blue text color, no padding, and no hover effect) — it's a styling prop not a functional one, i.e. it's not required for button links. When we remove it, we no longer need to restyle the button as we do below for `OpenWidgetViewerButton`.